x-pack/plugins/triggers_actions_ui/public/application/connectors_app.tsx

```diff
@@ -75,26 +73,21 @@ export const renderApp = (deps: TriggersAndActionsUiServices) => {
 };
 
 export const App = ({ deps }: { deps: TriggersAndActionsUiServices }) => {
-  const { dataViews, theme } = deps;
-  const isDarkMode = theme.getTheme().darkMode;
+  const { dataViews, i18n, theme } = deps;
   const sections: Section[] = ['connectors', 'logs'];
   const sectionsRegex = sections.join('|');
 
   setDataViewsService(dataViews);
   return (
-    <I18nProvider>
```

Comment:
The i18n context provider is not explicitly needed since KibnaRenderContextProvider provides it